src/python/system/environment.py

```diff
@@ -913,6 +913,9 @@ def set_bot_environment():
                                          'fuzzer-common-data-bundles')
   os.environ['IMAGES_DIR'] = os.path.join(inputs_dir, 'images')
   os.environ['SYMBOLS_DIR'] = os.path.join(inputs_dir, 'symbols')
+  os.environ['KERNEL_SYMBOLS_DIR'] = os.path.join(inputs_dir, 'symbols',
```

Comment:
It is just used in one place, so best to not define an environment var for it, but just add it locally in that function.